image/rhel/konflux.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
```suggestion
# TODO(ROX-20312): we can't pin image tag or digest because currently there's no mechanism to auto-update that.
FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```